.github/CODEOWNERS

```diff
@@ -362,8 +363,14 @@
 /logzio/*metadata.csv                    @DataDog/agent-integrations @DataDog/documentation
 /logzio/manifest.json                    @DataDog/agent-integrations @DataDog/documentation
 /logzio/README.md                        @DataDog/agent-integrations @DataDog/documentation
+<<<<<<< HEAD
```

Comment:
@iliakur there were merge conflicts in this file -- Do we need all 5 of these lines?